src/wallet/coin_selection.rs

```diff
@@ -118,9 +128,20 @@ pub struct CoinSelectionResult {
     pub selected: Vec<Utxo>,
     /// Total fee amount in satoshi
     pub fee_amount: u64,
+    /// Waste value of current coin selection
+    pub waste: Waste,
 }
 
 impl CoinSelectionResult {
+    /// Create new CoinSelectionResult
+    pub fn new(selected_utxos: Vec<Utxo>, fee_amount: u64, selection_waste: Waste) -> Self {
+        CoinSelectionResult {
+            selected: selected_utxos,
+            fee_amount,
+            waste: selection_waste,
+        }
+    }
+
```

Comment:
Not really needed, we can just create directly `CoinSelectionResult` (as we were doing before)